res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -62,6 +56,13 @@ var intervalsPerRev = 1200,
     rpm = 33 + 1 / 3,  //Like a real vinyl !!! :)
     alpha = 1.0 / 8,   //Adjust to suit.
     beta = alpha / 32; //Adjust to suit.
+    
+/**************************
+ * Loop Size array 
+ * first 4 values used for Autoloop Not shifted
+ * last 4 values used for Autoloop Shifted
+ **************************/    
+var loopsize = [1, 2, 4, 8, 0.125, 0.25, 0.5, 16];
```

Comment:
The way I coded it makes it really easier for anyone to adjust as they wish in the config. PCDJ has it as 1/, 1, 4 and 16 not shifted so I tend to think that it is a matter of personal preference. I'm OK to commit as you say,